packages/app/features/Invitation/index.tsx

```diff
@@ -74,8 +74,14 @@ export const InvitationFeature: FC<Props> = ({
 
 	return (
 		<View style={[styles.container, style]}>
-			<InvitationHeader logoSrc={logoSrc} logoSize={logoSize} />
-			<View style={styles.commandContainer}>
+			<View />
```

Comment:
I guess we need this?

Reply:
this `<View />` is served as the empty header for `justifyContent: space-between`